src/context/Authentication/index.js

```diff
@@ -116,6 +116,8 @@ const AuthProvider = ({ children }) => {
         customUserData,
         previousAction,
         setPreviousAction,
+        setCustomUserData,
+        updateCustomUserData
```

Comment:
Hier könnte ich mir vorstellen, dass du setCustomUserData gar nicht zwingend übergeben musst, nur um es dann später als Param für updateCustomUserData zu verwenden. Z.b. signUserOut müsstest du hier eigentlich auch noch übergeben, da du es später als Param verwendest. Alles Parameter, die du in der Funktion updateCustomUserData übergeben musst, stammen sowieso aus dem Context. Also vielleicht stattdessen:
`updateCustomUserData: () => updateCustomUserData(isAuthenticated, token, setCustomUserData, userId, signUserOut)`

Ich kommentiere dann noch an der anderen Stelle, wo du die Parameter entfernen kannst.

Reply:
Okay, probier ich gleich mal aus. :)